### ISSUE TITLE
Is this repo maintained?

### ISSUE THREAD

**Issue by amok** [cloudflare/pages-plugins#40]
It appears that the issues are not being handled effectively or addressed in a timely manner. Cloudflare, are there any plans to resolve these issues?

**Comment by nunoarruda** [comment]
Yeah, I've tried using [@cloudflare/pages-plugin-sentry](https://developers.cloudflare.com/pages/functions/plugins/sentry/) today, and found several issues.

@GregBrimble, can you please provide any feedback when you have a chance?

**Comment by motin** [comment]
No this repo is not maintained. Cloudflare has also chosen to deprecate the Pages platform in favor of Cloudflare Workers (https://developers.cloudflare.com/workers/static-assets/migration-guides/migrate-from-pages/), so don't expect any updates here. Find other packages, many of the original vendor plugins may work with Cloudflare Workers nowadays.
